business_objects/record_label_association.py

```diff
@@ -199,6 +199,28 @@ def get_all_classifications_for_information_source(
     )
 
 
+def get_existing_record_label_associations(
```

Comment:
name suggests all but we only collect manual rlas

Reply:
renamed